models/believer/believer.sdf

```diff
@@ -0,0 +1,693 @@
+<sdf version='1.6'>
+  <model name='believer'>
+    <pose>0 0 0.246 0 0 0</pose>
+    <link name='base_link'>
+      <pose>0 0 0 0 -0 0</pose>
+      <inertial>
+        <pose>0.15 0 0.1 0 -0 0</pose>
+        <mass>2.05</mass>
+        <inertia>
+          <ixx>0.16632</ixx>
+          <ixy>0</ixy>
+          <ixz>0.0755</ixz>
+          <iyy>0.3899</iyy>
+          <iyz>0</iyz>
+          <izz>0.5243</izz>
+        </inertia>
+      </inertial>
+      <collision name='base_link_collision'>
+        <pose>0 0 -0.07 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>1.088 2.591 0.103</size>
```

Comment:
wondering if you could use the a downsized STL as the collision model.

Reply:
We could, but I am not sure what value this would bring. Currently the box allows you to smoothly takeoff from the ground :)